docs/winui3_migration.md

```diff
@@ -18,4 +18,4 @@ This porting assistance is provided in the form of Roslyn Analyzers and Code Fix
 - A Code Analyzer provides on the fly code inspections for C# and creates diagnostics. 
 - A Code Fix consumes the diagnostics created by the Code Analyzer and modifies the C# file in-line. 
 
-See the [try-convert documentation](https://github.com/dotnet/try-convert/blob/feature/winui) for more information on these code analyzers and how to automoate the process with [try-convert](https://github.com/dotnet/try-convert/blob/feature/winui/WinUIConvert.md).
\ No newline at end of file
+See the [try-convert documentation](https://github.com/dotnet/try-convert/blob/feature/winui/WinUIConvert.md) for more information on these code analyzers and how to automoate the process with try-convert.
```

Comment:
The link in "It is recommended to use the try-convert tool for the full conversion process. try-convert will automate most of the conversion steps.", i.e. https://github.com/dotnet/try-convert/blob/feature/winui is still broken in the beginning of the file.

Reply:
Thanks for the catch! I think all the links are good now.

Reply:
See #6391